calculator/src/main/java/ru/neoflex/calculator/controller/CalculatorController.java

```diff
@@ -0,0 +1,29 @@
+package ru.neoflex.calculator.controller;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.List;
+
+import org.springframework.web.bind.annotation.*;
+
+import lombok.AllArgsConstructor;
+import ru.neoflex.calculator.dto.*;
+import ru.neoflex.calculator.service.ValidationService;
+
+@RestController
+@RequestMapping("/calculator")
+@AllArgsConstructor
+public class CalculatorController {
+    ValidationService validationService;
```

Comment:
нет модификатора доступа
как вариант:
```suggestion
private final ValidationService validationService;
```